app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,36 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: [:create, :destroy]
+  before_action :correct_user, only: :destroy
+
+  # creating the post if user is log in
+  # user/micropost/new
```

Comment:
Please mention the CORRECT route, alongwith HTTP method too.

Same applies in other comment too.

Please update that.